lib/client/api.go

```diff
@@ -1116,10 +1116,7 @@ func NewClient(c *Config) (tc *TeleportClient, err error) {
 		return nil, trace.BadParameter("No proxy address specified, missed --proxy flag?")
 	}
 	if c.HostLogin == "" {
-		c.HostLogin, err = Username()
-		if err != nil {
-			return nil, trace.Wrap(err)
-		}
+		c.HostLogin = c.Username
```

Comment:
If users specify the Username but not the HostLogin, they get different behavior.
Previously, it would set HostLogin to the result of `Username()`.
Now, it will be the specified Username.

I think it's fine to default to Username if specified, just wanted to point out the change in behavior.